Manual.md

```diff
@@ -364,8 +364,8 @@ The list of mandatory variables for a template:
 
 - `homepage` A string pointing to the `upstream` homepage.
 
-- `license` A string matching any license file available in `/usr/share/licenses`.
-Multiple licenses should be separated by commas, Example: `GPL-3, LGPL-2.1`.
+- `license` A string with the license's [SPDX identifier](https://spdx.org/licenses/).
```

Comment:
problably needs rewording

Reply:
Sure. Suggestions welcome.

Reply:
I'm not a big fan of starting with 'A <thing>' i prefer putting only '<thing>' but since the other variables do it lets keep it uniform.

Something along the lines of

```
A string matching the license's [SPDX Short identifier](https://spdx.org/licenses)
```

Reply:
Sounds good to me. Changed.